cmake/set_compile_flags.cmake

```diff
@@ -18,12 +18,12 @@ endif()
 # Add our extra flags according to language
 separate_arguments(AMR_WIND_CXX_FLAGS)
 target_compile_options(
-  ${amr_wind_lib_name} PUBLIC
+  ${amr_wind_lib_name} PRIVATE
```

Comment:
Just want to double check this is intentional.

Reply:
@jrood-nrel Yes. Currently this is all warning flags and it is leaking those flags to downstream. I suppose we should have a way to separate out `warning` flags from other flags.